### ISSUE TITLE
Flags to disable certain checks in Settings is ignored

### ISSUE THREAD

**Issue by Karunamon** [AtomLinter/linter-bootlint#20]
Greetings,

I'm having trouble getting specific lint checks to be ignored using the Atom settings page. Given the following snippet of HTML:

```html
  <div class="form-group">
            <label for="sizeinmb" class="col-sm-2 control-label">Size (MB)</label>
            <div class="col-sm-10">
              <input class="form-control" type="num" id="sizeinmb" />
            </div>
          </div>
````

An E042 code is generated. This is correct (as the type should be "number" rather than "num"), however, entering E042 in the Flags section of the configuration does not suppress the message.

**Comment by Arcanemagus** [comment]
Finally getting a chance to look at this, and it is definitely a bug, looking into the cause.

**Comment by Arcanemagus** [comment]
Fix published in v1.0.2, thanks for reporting this!
